third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -215,7 +230,8 @@ func resourceComputeInstanceGroupManager() *schema.Resource {
 
 <% if version.nil? || version == 'ga' -%>
 			"rolling_update_policy": &schema.Schema{
-				Deprecated: "This field is in beta and will be removed from this provider. Use it in the the google-beta provider instead. See https://terraform.io/docs/providers/google/provider_versions.html for more details.",
+				Removed:    "This field is in beta. Use it in the the google-beta provider instead. See https://terraform.io/docs/providers/google/provider_versions.html for more details.",
+				Computed:   true,
```

Comment:
Hmm, why do we need to set this to computed?

Reply:
Afaik I got the impression at least once that you needed to do that in some edge case while removing some fields to avoid a diff, and have done so on any removal I've done since. Do you think we shouldn't have it set? Imo it's mostly harmless and as far as I can recall can only help.